message/listener_test.go

```diff
@@ -13,16 +14,16 @@ import (
 
 var messagesProcessed = 0
 
-func mockFilterFunc(filterRequest handlers.FilterRequest) handlers.FilterResponse {
+func mockFilterFunc(filterRequest event.FilterRequest) handlers.FilterResponse {
 	messagesProcessed++
 	return handlers.FilterResponse{Message: "done"}
 }
 
 func TestProcessor(t *testing.T) {
-	event := handlers.FilterRequest{
-		InputFilePath:  "inputUrl",
-		OutputFilePath: "outputUrl",
-		Dimensions: map[string][]string{
+	event, _ := event.NewFilterRequest(
```

Comment:
It's test code, so we can be pretty certain there won't be any error at this point